src/pybind/mgr/nfs/export.py

```diff
@@ -659,7 +660,13 @@ def create_cephfs_export(self,
                              clients: list = [],
                              sectype: Optional[List[str]] = None) -> Dict[str, Any]:
 
-        check_cephfs_path(self.mgr, fs_name, path)
+        try:
+            check_cephfs_path(self.mgr, fs_name, path)
+        except cephfs.ObjectNotFound:
+            raise NFSObjectNotFound(f"either path {path} does not exist or"
+                                    " it might be a symlink which is not allowed")
+        except (cephfs.Error, NotADirectoryError) as e:
```

Comment:
Its not 100% clear to me how moving the exception handling to the caller fixes the redundant message in error string. Could you explain?

Reply:
Is that due to `raise ErrorResponse.wrap(e)` in create_export?

Reply:
earlier it was raised within the helper itself https://github.com/ceph/ceph/pull/49460/files#diff-66d1478f0663fee84e457bafb5b1fe683ad3701c4a30f92412e7df9a8acf5cca

so i dont think it should be this part of code right?